android/festago/app/src/main/java/com/festago/festago/presentation/ui/ticketentry/TicketEntryViewModel.kt

```diff
@@ -38,6 +45,13 @@ class TicketEntryViewModel(
         }
     }
 
+    fun loadTicket(ticketId: Long) {
+        viewModelScope.launch {
+            val ticket = ticketRepository.loadTicket(ticketId)
+            _ticket.value = ticket.toPresentation()
```

Comment:
```suggestion
            _ticket.value = ticketRepository.loadTicket(ticketId).toPresentation()
```
저는 지역변수를 사용하는 이유는 이해를 돕기 위하거나 값을 저장하기 위해서라고 생각합니다!
해당 코드에서 받아오는 함수명이 `loadTicket` 이고 결과 값을 받는 변수가 `_ticket.value`이기에 `val ticket`으로 하지 않고 바로 넣어도 이해하는데에 무리가 없다고 생각합니다!